pandas_gbq/gbq.py

```diff
@@ -15,6 +15,8 @@
 from pandas.core.common import PandasError
 from pandas.compat import lzip, bytes_to_str
 
+logger = logging.getLogger(__name__)
```

Comment:
I noticed the following message when running unit tests
```
No handlers could be found for logger "pandas_gbq.gbq"
```

Could we add a NullHandler to prevent the error when no logging configuration is provided? From the [logging docs](https://docs.python.org/2/howto/logging.html#configuring-logging-for-a-library),

> A do-nothing handler is included in the logging package: NullHandler (since Python 2.7). An instance of this handler could be added to the top-level logger of the logging namespace used by the library (if you want to prevent an error message being output to sys.stderr in the absence of logging configuration). 

For example,
```
logger = logging.getLogger(__name__)
logger.addHandler(logging.NullHandler())
```

On a separate note, could we enable info level logging by default when running tests (in test_gbq.py)?